receiver/snmpreceiver/config.go

```diff
@@ -136,7 +136,9 @@ type Config struct {
 
 	// Metrics defines what SNMP metrics will be collected for this receiver and is composed of metric
 	// names along with their metric configurations
+	// TODO hughesjj wait wat this doesn't look right
 	Metrics map[string]*MetricConfig `mapstructure:"metrics"`
+	//metadata.MetricConfig `mapstructure:"metrics"`
```

Comment:
This receiver is different, it doesn't have a predefined list of metrics. Please ignore it